### PR TITLE
Remove python38 typing

### PR DESCRIPTION
**Pull request by Jokilos**

### CODE REVIEW

**Review by lukaszsmolinski — APPROVED** [review]

File: midi_app_controller/gui/binds_editor.py [hunk 1]
```diff
@@ -227,7 +225,7 @@ class BindsEditor(QDialog):
 
     Attributes
     ----------
-    save_binds : Callable[[List[KnobBind], List[ButtonBind]], None]
+    save_binds : callable[[list[KnobBind], list[ButtonBind]], None]
```
Comment (Aleshkev) [inline midi_app_controller/gui/binds_editor.py:228]: there is no callable[...] generic!! You still have to do `from typing import Callable` and use `Callable`.

See e.g. https://stackoverflow.com/questions/66902082/did-python-3-9-update-how-to-type-hint-the-function-type

File: midi_app_controller/models/utils.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def save_to(self, path: Path) -> None:
             yaml.safe_dump(self.dict(), f, default_flow_style=False)
 
 
-def find_duplicate(values: List[Any]) -> Optional[Any]:
+def find_duplicate(values: list[any]) -> Optional[any]:
```
Comment (Aleshkev) [inline midi_app_controller/models/utils.py:63]: the same with `any`, i think the only builtin functions that have become aliases for generics are list, tuple, dict, set, frozenset and type

**Review by Aleshkev — APPROVED** [review]
maybe these `any` and `callable` should be fixed before merging?

**Comment by Aleshkev** [comment]
Also, maybe we should consider using something like `mypy` to check if the types are actually correct

**Review by modyf01 — APPROVED** [review]

**Pull request merged**